tests/static-analysis/random.php

```diff
@@ -38,27 +38,19 @@ function random_checkStringMap(CollectionInterface $collection): void
 }
 
 $intGenerator = static function (): Generator {
-    while (true) {
```

Comment:
for now `while (true)` is a new PHPStan error: https://github.com/phpstan/phpstan/issues/5917.

These ones don't matter for us though, makes no difference for the static analysis checks

Reply:
Nice !